README.md

```diff
@@ -6,17 +6,39 @@
 
 ## What is Rust?
 
-At a high level Rust is a language thats goal is to allow programmers to get systems-level performance without devoting the time to learn about manual memory managment. It also comes out of the box with a set of tools to steer you towards writing better, more optomized code.
+At a high level Rust is a language thats goal is to allow programmers to get systems-level performance without devoting the time to learn about manual memory management. It also comes out of the box with a set of tools to steer you towards writing better, more optimized code.
+
+## Installing Rust
+
+```sh
+curl https://sh.rustup.rs -sSf | sh  
+source $HOME/.cargo/env
+rustup install nightly 
+```
 
 ## Where to Start
+
+The Docs! Rust can build an offline copy of the documentation and it is strongly encouraged that you start here to learn about the capabilities and the built in methods of Rust.
+
+```sh
+rustup run nightly cargo doc --open 
```

Comment:
This is to build this particular project, but I think that given the above paragraph, we should first include `rustup docs`, which seems more what you’re talking about

Reply:
Nice, I figured that they were the same, but `rustup docs` is nice distinction